packages/tree-sitter/build.sh

```diff
@@ -16,15 +17,26 @@ termux_step_pre_get_source() {
 	local _SOVERSION=0.24
 
 	# New SO version is the major version of the package
-	if [[ $TERMUX_PKG_VERSION != $_SOVERSION.* ]]; then
+	if [[ "$TERMUX_PKG_VERSION" != "${_SOVERSION}".* ]]; then
 		termux_error_exit "SOVERSION guard check failed."
 	fi
 }
 
-termux_step_post_make_install() {
+termux_step_pre_configure() {
 	termux_setup_rust
+	unset CFLAGS
```

Comment:
Could you please clarify the reason for this requirement?

Reply:
Mmmh yep good idea.
This is something @truboxl mentioned to me a while ago.
The CFLAGS interfere with the host build section on 32bit builds.
https://github.com/termux/termux-packages/pull/22508#issuecomment-2533467997